crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -41,29 +40,15 @@ pub struct Logger {
 impl ExtensionFactory for Logger {
     fn create(&self) -> Arc<dyn Extension> {
         Arc::new(LoggerExtension {
-            session_id: "".to_string().into(),
             config: self.config.clone(),
         })
     }
 }
 
 struct LoggerExtension {
-    session_id: Mutex<String>,
     config: LoggerConfig,
 }
 
-impl LoggerExtension {
```

Comment:
We don't need these anymore, as we're using directly the context.
`ctx.data_unchecked::<SocketAddr>()`